### PR TITLE
adds Mon keynote title, abstract, and bio

### PR DESCRIPTION
**Pull request by cc10512**

### CODE REVIEW

File: _includes/keynotes.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+<h1>Keynotes</h1>
+
+<h2><a name="keynote1">Concurrent Data Sketches</a>, Idit Keidar</h2>
```
Comment (fadeopolis) [inline _includes/keynotes.html]: I'd add the date here too, just to be extra clear.
The table with the program is a bit far, and people may have to scroll back and forth when they are skimming the keynote abstracts and the dates.

File: program/index.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ show_sidebar: true
 			8:30
 		</td>
 		<td>
-			Keynote: Idit Keidar, Technion - Israel Institute of Technology
+			Keynote: <a href="index.html#keynote1">Concurrent Data Sketches</a>, Idit Keidar, Technion - Israel Institute of Technology
```
Comment (fadeopolis) [inline program/index.md]: Minor nitpick: `href="#keynote1"` would have the same effect here.

**Review by fadeopolis — APPROVED** [review]
Good to merge as is.
I'd personally redundantly add the dates in keynotes.html, but don't have a strong preference here.

File: program/index.md (hunk 2 above)
Comment (cc10512) [reply program/index.md]: done

File: _includes/keynotes.html (hunk 1 above)
Comment (cc10512) [reply _includes/keynotes.html]: ack. done

**Pull request merged**